openpype/modules/sync_server/sync_server_module.py

```diff
@@ -913,7 +916,42 @@ def reset_timer(self):
 
             In case of user's involvement (reset site), start that right away.
         """
-        self.sync_server_thread.reset_timer()
+
+        if not self.enabled:
+            return
+
+        if self.sync_server_thread is None:
+            self._reset_timer_with_rest_api()
+        else:
+            self.sync_server_thread.reset_timer()
+
+    def is_representaion_on_site(
+        self, project_name, representation_id, site_id
+    ):
+        # TODO implement
```

Comment:
Am I supposed to implement it or to wait before merging?

Reply:
We're working on it :)